### ISSUE TITLE
Performance lower than reported

### ISSUE THREAD

**Issue by chenz97** [dmis-lab/hats#4]
Hi, thank you for your great work and code! 

I ran your training code of node classification by `make test_phase=1 save_dir=save`, but have found that the performance on the validation set is relatively low at accuracy : 0.4146, hit ratio : 0.5090, macro f1 : 0.2762, expected return : -0.0087, which seems to be much lower than the figures reported in the paper.

Is there any configurations that should be modified to reproduce your results? Thank you very much!

**Comment by 280777510** [comment]
> Hi, thank you for your great work and code!
> 
> I ran your training code of node classification by `make test_phase=1 save_dir=save`, but have found that the performance on the validation set is relatively low at accuracy : 0.4146, hit ratio : 0.5090, macro f1 : 0.2762, expected return : -0.0087, which seems to be much lower than the figures reported in the paper.
> 
> Is there any configurations that should be modified to reproduce your results? Thank you very much!
Hello, I cannot download the stock data and Wikipedia relationship data provided in the original paper, could you please give me a copy?

**Comment by Mahdiehdn** [comment]
Hi,
The code of node classification does not run.
There is no file ordered_ticker.pkl in folder relation.

In class StockDataset/def read(self, )/
with open(os.path.join(self.rel_data_dir, 'ordered_ticker.pkl'), 'rb') as f:

What file should I use instead of 'ordered_ticker.pkl'?
